src/test/java/org/folio/edge/connexion/MainVerticleTest.java

```diff
@@ -35,17 +40,41 @@ public void instanceMainVerticvle() {
     Assert.assertEquals(100, v.getMaxRecordSize());
   }
 
-  @Test
-  public void testAdminHealth(TestContext context) {
-    final int port = 9230;
+  Future<Void> deploy(MainVerticle mainVerticle) {
     JsonObject config = new JsonObject();
-    config.put(SYS_PORT, port);
+    config.put(SYS_PORT, PORT);
+    return vertx.deployVerticle(mainVerticle, new DeploymentOptions().setConfig(config))
+        .mapEmpty();
+  }
 
+  @Test
+  public void testAdminHealth(TestContext context) {
     WebClient webClient = WebClient.create(vertx);
-    webClient.get(port, "localhost", "/admin/health").send().onComplete(context.asyncAssertFailure(f ->
-        vertx.deployVerticle(new MainVerticle(), new DeploymentOptions().setConfig(config))
-            .compose(x -> webClient.get(port, "localhost", "/admin/health").send())
+    deploy(new MainVerticle())
+        .compose(x -> webClient.get(PORT, "localhost", "/admin/health").send())
+        .onComplete(context.asyncAssertSuccess());
+  }
+
+  @Test
+  public void testImportWithUserOK(TestContext context) {
+    NetClient netClient = vertx.createNetClient();
+    deploy(new MainVerticle())
+        .compose(x -> netClient.connect(PORT, "localhost")
```

Comment:
```suggestion
        .compose(x -> vertx.createNetClient().connect(PORT, "localhost")
```
The netClient variable is no longer needed.